test/models/etmpToUseranswers/TransformationSpec.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
```

Comment:
There don't seem to be any tests in here

Reply:
This is a supporting class for TransformationSpecs. There is no such file in app so there can't be any tests in it

Reply:
I see, that's fine then